Modules/database_manager.py

```diff
@@ -11,19 +11,12 @@
 This module is built on top of the Pydle system.
 """
 import pyodbc
-import config
+from config import config
 import logging
 
-log = logging.getLogger(f"mecha.{__name__}")
-
-
-class Singleton(type):
-    _instances = {}
+from utils.ratlib import Singleton
 
-    def __call__(cls, *args, **kwargs):
-        if cls not in cls._instances:
-            cls._instances[cls] = super(Singleton, cls).__call__(*args, **kwargs)
-        return cls._instances[cls]
+log = logging.getLogger(f"mecha.{__name__}")
 
 
 # noinspection SqlNoDataSourceInspection

```

Comment:
I must ask why do we need a metaclass?
to quote Tim Peters: ```Metaclasses are deeper magic than 99% of users should ever worry about. If you wonder whether you need them, you don’t (the people who actually need them know with certainty that they need them, and don’t need an explanation about why).```

Reply:
Thanks for handing me quote I already quoted. Secondly: Show me a cleaner way to achieve that, while not having similar code floating around